backend/posts/views/votes.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth.models import User
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from posts.models import PostVote, CommentVote
+from posts.serializers import PostVoteSerializer, CommentVoteSerializer
+
+
+def _get_votes(request, queryset, serializer_type, vote_type):
+    if vote_type is not None:
```

Comment:
Funny that you are providing `queryset` and additional flag that filters this queryset for you `vote_type`. 
Maybe you should just stick to one of those 😂 ?

Reply:
Yeah, well, I wanted to make it better and it ended up like this xd

At first, this filter was in `post_votes` and in `comment_votes` separately but then PyCharm was shouting at me that I had repetitions, so I moved it into `_get_votes` and it's like this.

I also can't move the whole queryset to the `_get_votes` because I'd need to pass even more parameters to specify the model and filter params.

I guess I'll just cope with the repetitions then.